src/APNS-Topics.erl

```diff
@@ -0,0 +1,268 @@
+%% Generated by the Erlang ASN.1 BER_V2-compiler version, utilizing bit-syntax:4.0.4
+%% Purpose: encoder and decoder to the types in mod APNS-Topics
+
+-module('APNS-Topics').
```

Comment:
Do we need this file in the repo? As far as I understand it's generated before compilation.